CesiumGltf/src/Model.cpp

```diff
@@ -208,3 +211,366 @@ void Model::merge(Model&& rhs) {
     this->scene = int32_t(this->scenes.size() - 1);
   }
 }
+
+namespace {
+void forEachPrimitiveInMeshObject(
```

Comment:
Those functions are quite useful. I can imagine you can replace the Unreal gltf loadNode code and the like with those foreach